.github/workflows/performance.yml

```diff
@@ -0,0 +1,111 @@
+
+# Schedule triggers
+on:
+  # TODO this is just while developing
+  pull_request:
+    branches:
+      - 'develop' 
+      - 'performance-regression-testing'
+  schedule:
+    # runs every day at 10:05pm
+    - cron:  '5 22 * * *'
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+
+  test-rust:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+      - uses: actions-rs/cargo@v1
+        with:
+          command: test
+          args: --manifest-path performance/runner/Cargo.toml
+
+  build-rust:
+    needs: [test-rust]
+    name: Build Rust
+    runs-on: ubuntu-latest
+    env:
+      RUSTFLAGS: "-D warnings"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+      - uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --release --manifest-path performance/runner/Cargo.toml
+      - uses: actions/upload-artifact@v2
+        with:
+          name: runner
+          path: performance/runner/target/release/
+
+  measure-develop:
+    needs: [build-rust]
+    name: Measure develop Branch
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/download-artifact@v2
+        with:
+          name: runner
+      - name: checkout develop
+        uses: actions/checkout@v2
+        with:
+          ref: 'develop'
```

Comment:
Maybe we don't specify `develop` here and instead let schedule runs use `develop` and then also allow us to pick the branch we want in manual runs. We can have the scheduled run default to `develop`. With manual runs you can pick a branch and then it will default checkout that

Reply:
that sounds great! .....how do you write that down in a workflow?